stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatViewModel.kt

```diff
@@ -21,13 +23,14 @@ class ChatViewModel(private val cid: String, private val chatDomain: ChatDomain
         }
     }
 
+    @OptIn(InternalStreamChatApi::class)
```

Comment:
I'm not a big fan of that. Our sample app shouldn't use things that should be unavailable for our customers. I would rather move `ChannelData::isDistinctChannel` extensions directly to the sample app